integration/internal/container/ops.go

```diff
@@ -142,3 +142,13 @@ func WithAutoRemove(c *TestContainerConfig) {
 	}
 	c.HostConfig.AutoRemove = true
 }
+
+// WithPidsLimit sets the container's "pids-limit
+func WithPidsLimit(limit *int64) func(*TestContainerConfig) {
```

Comment:
Was in doubt if I should change this to a non-pointer to make it easier to use (and if no limit is needed, don't pass `WithPidsLimit()`)

Reply:
I can see both ways here. In the end I think making a pointer is simpler than dealing with filtering options... e.g.


```go
opts := []ConfigOpts{WithFoo,WithBar}
if hasLimit {
  opts = append(opts, WithPidsLimit(limit))
}
container.Run(..., opts...)
```